metron-platform/metron-common/src/main/scripts/stellar

```diff
@@ -28,10 +28,15 @@ elif [ -e /usr/lib/bigtop-utils/bigtop-detect-javahome ]; then
   . /usr/lib/bigtop-utils/bigtop-detect-javahome
 fi
 
+export METRON_SYSCONFIG="/etc/default/metron"
+if [ -f "$METRON_SYSCONFIG" ]; then
+	source $METRON_SYSCONFIG
+fi
+
+export METRON_VERSION="${METRON_VERSION:-${project.version}}"
+export METRON_HOME="${METRON_HOME:-/usr/metron/$METRON_VERSION}"
```

Comment:
Even though we source `/etc/default/metron` which does define `METRON_HOME` and `METRON_VERSION`, I chose to continue to set a value for these should `/etc/default/metron` not exist.